editor/src/components/canvas/canvas-strategies/strategies/draw-to-insert-metastrategy.tsx

```diff
@@ -303,6 +303,7 @@ export function drawToInsertStrategyFactory(
                   resizeCommand,
                   ...optionalWrappingCommand,
                 ],
+                'rerender-all-elements',
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.